### DISCUSSION TITLE
启用代理无网络，unblockneteasemusic

### DISCUSSION THREAD

**Discussion by shanxin567** [qier222/YesPlayMusic#1163] [Q&A]
问题描述：
在docker上面部署了unblockneteasemusic服务，使用原版网易云音乐客户端开启代理功能正常使用，可以解锁灰色歌曲（此时开启clash代理）。
使用YesPlayMusic开启代理功能后，软件无法连接网络，开启和关闭clash代理软件时都无发连接网络，浏览器可以正常访问网络。

### 系统信息
操作系统：win10
软件版本：0.4.2安装版
![Snipaste_2022-01-02_22-25-13](https://user-images.githubusercontent.com/58086406/147878855-52719887-211d-4e23-b2c7-399b24a8b07c.png)

**Comment by pan93412** [comment]
YPM 已經內建 UnblockNeteaseMusic，不需要手動設定代理。

**Reply by shanxin567** [reply]
该代理功能是否已经失去的它该有的效果，是否需要在开启YOM内部UnblockNeteaseMusic功能时，隐藏代理功能，在关闭内部UnblockNeteaseMusic功能时，可以选择启用代理功能呢，被代理问题折磨一晚上，一直以为是自己docker服务器的问题

**Reply by pan93412** [reply]
> 该代理功能是否已经失去的它该有的效果，是否需要在开启YOM内部UnblockNeteaseMusic功能时，隐藏代理功能，在关闭内部UnblockNeteaseMusic功能时，可以选择启用代理功能呢，被代理问题折磨一晚上，一直以为是自己docker服务器的问题

原本設計代理功能的使用場景是用來播放海外音樂（比如播放 QQ 音樂海外版 JOOX 的音樂），或者是越過防火牆的阻擋規則。我們沒辦法判斷對方是不是 UNM Proxy 伺服器，所以也沒辦法做類似的防誤操作機制。
